examples/minimal/outputs.tf

```
@@ -0,0 +1,7 @@
+output "ingress_ports" {
+  value = concat(
+  var.ports,
+  var.additional_ports,
+  )
```

Why not just return the outputs of the modules?
```suggestion
  value = module.sg-ports
```

---

Done